test/elementhandle.spec.ts

```diff
@@ -49,7 +49,7 @@ describe('ElementHandle specs', function () {
       const elementHandle = await nestedFrame.$('div');
       const box = await elementHandle.boundingBox();
       if (isChrome)
-        expect(box).toEqual({ x: 28, y: 260, width: 264, height: 18 });
+        expect(box).toEqual({ x: 28, y: 182, width: 264, height: 18 });
       else expect(box).toEqual({ x: 28, y: 182, width: 254, height: 18 });
```

Comment:
Now these two `expect` calls are the same, I think we can lose the conditional.

Are we confident that this change is expected?

Reply:
The `width` is different though?